.changeset/tricky-shoes-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-graph': minor
+---
+
+Added Action attribute for CatalogGraphCard
```

Comment:
Could you add an example here explaining briefly how to use the new property?

Reply:
it's a property from a core Card component. Does it make a sense to do it here?

Reply:
This description is added to the plugin changelog file, so people are not necessarily aware how the CatalogGraphCard is implemented and that it forwards the action property to an InfoCard component under the hood. Because the changelog serves as a documentation for the new features released for this plugin, I feel it makes sense to include an example of what this property expects or to at the very least refer to the InfoCard documentation and say that the action property expects the same value as the InfoCard action property.
So I suggest adding an example like this, but with a real action component:
```tsx
const action =  <add an example of a real action component here>
<CatalogGraphCard action={action} />
```

Reply:
ok, comment added